microsoft-edge/devtools-guide-chromium/speed/get-started.md

```diff
@@ -22,440 +22,480 @@ ms.date: 05/04/2021
+   ![The new Coverage report, now showing the CSS files](./images/coverage-report-css.png)
 
-   ![The Coverage report](../media/speed-glitch-tony-remix-updated-audits-performance-oppportunities-expanded-drawer-coverage-reloaded.msft.png)
+    The report shows that the `contact-form.css` and `gallery.css` files aren't used at all. They both have 100% of unused bytes.
 
-1. Click the `jquery.js` row.  DevTools opens the file in the **Sources** tool.  If a line of code ran, a blue bar appears next to it.  A red bar means the line of code was not run, and is definitely not needed on load of the webpage.
+1. Click the `contact-form.css` file in the report.  DevTools opens the file in the **Sources** tool.  If a line of code ran, a blue bar appears next to it.  A red bar means the line of code didn't run, and is definitely not needed on load of the webpage.
 
-   ![Viewing the jQuery file in the Sources tool](../media/speed-glitch-tony-remix-updated-sources-drawer-coverage-reloaded-jquery-js.msft.png)
+   ![The contact-form.css file in the Sources tool, with red bars next to the unused lines](./images/unused-css-source.png)
 
-1. Scroll through the jQuery code.  Some of the lines that run are actually just comments.  To strip comments and reduce the size of the file, run the code through a minifier app or script.
+   Only red bars are displayed in this source file, which means that the webpage does not need this file at all. 
 
-In short, when you're working with your own code, the **Coverage** tool helps you analyze your code, line-by-line, and only ship the code that's needed for page load.
+Now, remove the references to these files from the code:
 
-Are the `jquery.js` and `lodash.js` files even needed to load the page?  The **Request blocking** tool shows what happens when resources aren't available:
+1. In Visual Studio Code, open the `/travel-site/index.html` file.
 
-1. Select the **Network** tool.
+1. Near the top of the file, find the list of `<link>` tags that load the stylesheets on the page.
 
-1. Press `Ctrl`+`Shift`+`P` (Windows, Linux) or `Command`+`Shift`+`P` (macOS) to open the Command Menu again.
+1. Delete the two lines of code that load the `contact-form.css` and `gallery.css` files:
 
-1. Start typing `blocking`, and then select **Show Request Blocking**.
+    ![VS Code, with the index.html opened, showing where the two link tags are](./images/delete-unused-css.png)
 
-   ![The Request blocking tool](../media/speed-glitch-tony-remix-updated-network-drawer-request-blocking-empty.msft.png)
+1. Save your changes in Visual Studio Code, then refresh the webpage in Microsoft Edge, and run a new audit in the **Lighthouse** tool again to see how your changes affect the load performance.
 
-1. Click **Add Pattern** (![Add Pattern](../media/add-pattern-icon.msft.png)), type `/libs/*`, and then press `Enter` to confirm.
+###### Automatically remove non-critical CSS
```

Comment:
i'd have to check context - change from h5 to h4 or h6?  often pattern h2/h4/h6 works well for spread of sizes given current LMC css